services/api-server/src/simcore_service_api_server/services_rpc/wb_api_server.py

```diff
@@ -588,13 +596,15 @@ async def find_cached_function_jobs(
         user_id: UserID,
         product_name: ProductName,
         function_id: FunctionID,
-        inputs: FunctionInputs,
-    ) -> list[RegisteredFunctionJob] | None:
+        inputs: FunctionInputsList,
+        status_filter: list[FunctionJobStatus] | None,
```

Comment:
I would make this name more self-documenting. I 'assume' this is the statuses that are considered cached? Maybe cached_job_statuses or so?